codes/javascript/test_all.ts

```diff
@@ -0,0 +1,60 @@
+import { bold, brightRed } from 'jsr:@std/fmt/colors';
```

Comment:
Can we use a `.js` file instead? It's a little confusing to test JS code with a `.ts` file.